data/json/monsters/zed_misc.json

```
@@ -390,7 +390,7 @@
     "copy-from": "mon_zombie_base",
     "diff": 5,
     "volume": "875 L",
-    "weight": "200 kg",
+    "weight": "1000 kg",
```

```suggestion
    "weight": "1000 kg"
```
New commit to fire the new required clang test